extensions/wikia/AdEngine/js/template/floor.js

```diff
@@ -0,0 +1,67 @@
+/*global define*/
+define('ext.wikia.adEngine.template.floor', [
+	'jquery',
+	'wikia.log',
+	'wikia.document',
+	'wikia.iframeWriter',
+	'wikia.window'
+], function ($, log, doc, iframeWriter, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.template.floor',
+		footerHtml = '<div id="ext-wikia-adEngine-template-footer">' +
+			'<div class="background"></div>' +
+			'<div class="ad"></div>' +
+			'<a class="close" title="Close" href="#"><span>Close</span></a>' +
```

Comment:
It doesn't matter for us if it's English or Polish, does it? Everyone understands "Close" in advertisments :japanese_ogre: